src/modules/Core/Account/settings.js

```diff
@@ -196,6 +283,31 @@ export const setSyncedSettings = (account: EdgeAccount, settings: Object) => {
   SettingsFile.setText(text)
 }
 
+export const setIsCustomNodesEnabled = async (account: EdgeAccount, currencyCode: string, isEnabled: boolean) => {
```

Comment:
Wait... is the GUI trying to save this setting to disk? The core is already doing that. Just check `edgeAccount.currencyTools[plugin].settings`, and whatever you put in there the last time will still be there, even after logging out / syncing phones / etc.